db/schema.rb

```diff
@@ -1115,6 +1115,7 @@
     t.string "onboarding_variant_version", default: "0"
     t.boolean "org_admin", default: false
     t.integer "organization_id"
+    t.datetime "organization_info_updated_at", default: "2017-01-01 05:00:00"
```

Comment:
I noticed the schema still contains the default, sorry @seb1441 for not nothing it before! :-)

Reply:
@rhymes Fixed!